### DISCUSSION TITLE
Where to deploy Remix + Postgres app beside Fly.io

### DISCUSSION THREAD

**Discussion by lukejagodzinski** [remix-run/remix#9200] [Q&A]
Hey, I need your opinion on what is the best PaaS provider beside Fly.io. So far Fly.io was the best option in terms of quality vs. cost. However, there were a few problems with it that made me consider switching to some other platform. However, I don't want to make a switch right now, I would like to have a backup option to switch just in case. So, ideally I would like some other platform that could use the exact same code and no special adapters like Vercel or others. Just take the same code, add config to my deployment workflow and that's it.

The problem is that not many platforms offer hosting Postgres. I really want server and db to be close to each other or ideally on the same server. I've started configuring Google Cloud Run but there is no Postgres option where I only pay for the actual usage like on Fly.io. I'm considering configuring all the stuff on GCP manually and just pausing all the servers but that will be a lot of work and I'm not DevOp.

Are there any other good platforms similar to Fly.io where I could just pay very little if I'm not using the app and it's waiting there being idle for the right time to use?

**Comment by alexanderson1993** [comment]
I think http://railway.app covers what you're looking for. I was able to run Remix + Postgres for < $5/mo, and they let you enable app sleeping, where the app turns off if it's not used for a period of time.

**Reply by lukejagodzinski** [reply]
Ok thanks I will check them out

**Reply by tombohub** [reply]
they have app sleeping only for javascript apps or any? How's the speed? My django app was very slow
